.github/workflows/release.yaml

```
@@ -47,6 +49,11 @@ jobs:
 
           script/release
 
+      - name: Upload release assets
+        uses: softprops/action-gh-release@v1
+        with:
+          files: dist/*
```

```suggestion
          files: |
            dist/home_assistant_frontend-*.whl
            dist/home_assistant_frontend-*.tar.gz
```
Lets make this more spesific

---

That won't quite work. Only the wheel uses underscores whereas the sdist has dashes. They are the same for pip though. I would recommend `dist/*.whl` and `dist/*.tar.gz`.